
{
    "module": "mojo.tests.testplus.local.test_import_failure",
    "filename": "/Users/myron/repos/testplus/source/testroots/mojo/tests/testplus/local/test_import_failure.py",
    "trace": [
        "Traceback (most recent call last):",
        "  File \"/Users/myron/repos/testplus/source/packages/mojo/testplus/queries.py\", line 50, in collect_test_references",
        "    mod = import_by_name(modname)",
        "          ^^^^^^^^^^^^^^^^^^^^^^^",
        "  File \"/Users/myron/repos/testplus/.venv/lib/python3.12/site-packages/mojo/xmods/ximport.py\", line 37, in import_by_name",
        "    mod = importlib.import_module(modulename)",
        "          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^",
        "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/importlib/__init__.py\", line 90, in import_module",
        "    return _bootstrap._gcd_import(name[level:], package, level)",
        "           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^",
        "  File \"<frozen importlib._bootstrap>\", line 1387, in _gcd_import",
        "  File \"<frozen importlib._bootstrap>\", line 1360, in _find_and_load",
        "  File \"<frozen importlib._bootstrap>\", line 1331, in _find_and_load_unlocked",
        "  File \"<frozen importlib._bootstrap>\", line 935, in _load_unlocked",
        "  File \"<frozen importlib._bootstrap_external>\", line 994, in exec_module",
        "  File \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed",
        "  File \"/Users/myron/repos/testplus/source/testroots/mojo/tests/testplus/local/test_import_failure.py\", line 2, in <module>",
        "    import blah",
        "ModuleNotFoundError: No module named 'blah'"
    ]
}